### DISCUSSION TITLE
Need help with Social Authentication

### DISCUSSION THREAD

**Discussion by suparerkk** [saleor/saleor#8479] [Q&A]
I have looking through a lot of issue and discussions but i have not found any complete direction of how social authentication can be implemented with saleor anywhere..., can anyone guide and make this thread useful please....

**Comment by karolkielecki** [comment]
@suparerkk Hi! There is no official social authentication yet. But maybe this one will be helpful -- check it out: https://github.com/saleor/saleor/issues/5228#issuecomment-596449248
